src/Accelerators/NNPA/Conversion/ONNXToZHigh/RewriteONNXForZHigh.cpp

```diff
@@ -281,6 +282,67 @@ Type CreatePaddedXType(Value x, ArrayAttr pads) {
   return paddedType;
 }
 
+class SplitLargeMatMulPattern : public OpRewritePattern<ONNXMatMulOp> {
```

Comment:
I know it's early in development, please add a comment to explain the pattern using math mul tensors, that would be helpful for current and further reading of the code, tx

Reply:
Added a comment. Thanks!